setup.py

```diff
@@ -2,15 +2,17 @@
 """
 Setup for edx-django-sites-extensions package
 """
+import io
+
 from setuptools import setup
 
 
-with open('README.rst') as readme:
+with io.open('README.rst',  encoding='utf-8') as readme:
     long_description = readme.read()
```

Comment:
@jmbowman I've updated the mechanism of how the README is being added in the description.

Reply:
You should probably also add the `twine` validation check for the built package somewhere in the tests run in Travis, like [here](https://github.com/edx/edx-django-utils/blob/master/tox.ini#L62).  Sometimes the problem is with invalid reStructuredText markup rather than a text encoding problem, and without this check it's hard to know if an upload will work until we've already tagged a release.